src/main/scala/com/kinja/soy/SoyMacroImpl.scala

```diff
@@ -106,13 +120,6 @@ object SoyMacroImpl {
     } else
       applyParamImplicits
 
-    // Collect missing implicits then abort with an error message for each missing implicit.
-    val missingImplicits = inferredImplicits.collect {
-      case Implicit(_, t, impl, rec, _) if (impl == EmptyTree && !rec) => t
-    }
-    if (missingImplicits.nonEmpty)
-      c.abort(c.enclosingPosition, s"No implicit SoyWrites for ${missingImplicits.mkString(", ")} available.")
```

Comment:
Since type-checking occurs again after the macro expansion, failure to find implicits will be reported by the compiler itself and we don't need to do it here.